### ISSUE TITLE
Users may need to override Pax Exam default options [PAXEXAM-310]

### ISSUE THREAD

**Issue by ops4j-issues** [ops4j/org.ops4j.pax.exam2#443]
**[Harald Wellmann](https://ops4j1.jira.com/secure/ViewProfile.jspa?accountId=557058:22a27d04-788e-4f94-b3c8-ac0f72869ed0)** created **[PAXEXAM-310](https://ops4j1.jira.com/browse/PAXEXAM-310)** 

`PaxExamRuntime` has a hard-coded set of default options, including some Pax Exam "system bundles", which cannot be changed, at least not when using the JUnit driver.

While the defaults are sensible and helpful for getting started quickly, they can cause conflicts for advanced use cases.

org.ops4j.pax.logging.api may conflict with the user's choice of logging framework.<br /> org.apache.geronimo.specs.atinject may conflict with other providers of package javax.inject (e.g. Weld).<br /> org.osgi.compendium may cause package uses conficts with Equinox, which contains some of the compendium packages in its system bundle, unlike Felix.<br /> org.ops4j.pax.exam.rbc is not required when using the Native or Forked Container.

In short, there should be a way to suppress all defaults, because Pax Exam Core and Containers cannot possible anticipate all user scenarios.

---

**Fixed in:** 2.4.0.RC1
Votes: 0, Watches: 0

---
**Referenced issues**

**aggregation of:**
* #186 - Give users a chance to avoid Pax Logging [PAXEXAM-252]
